docs/conf.py

```diff
@@ -81,12 +81,12 @@ def run(self):
     "show-inheritance": True,
 }
 
+# Generate autosummary pages. Output should be set with: `:toctree: pythonapi/`
+autosummary_generate = ['Python-API.rst']
+
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
```

Comment:
The only change I would suggest is moving these lines back below the `templates_path` variable to keep the diffs smaller.

Reply:
@hayesall Thanks for your review! I moved `autosummary_generate` variable with the aim to keep logic grouping: `autodoc_*` variables are closer to `autosummary_*` and `templates_path` is closer to `source_suffix`, so the first group is about `auto_` extensions and the second one is about source file paths.

IDK, maybe you're right and smaller diff is more important than grouping. Let's wait for one more opinion from another reviewer...

Reply:
@StrikerRUS Logical ordering is more important small diffs, I like this now that I see it this way 👍 

While we're on the subject: it doesn't appear that templates are used in this repository, so we could drop the `templates_path` variable.

Reply:
@hayesall Good suggestion! Thanks! I also dropped commented option and reorganized two other options connected to extensions in the latest commit 98ebbd833410f8e3d9cd03522af23c60068c0a78.